pkgs/development/python-modules/declinate/default.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitLab,
+
+  # build-system
+  setuptools,
+  setuptools-scm,
+
+  # dependencies
+  typing-extensions,
+
+  # nativeCheckInputs
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "declinate";
+  version = "0.0.6";
+  pyproject = true;
+
+  src = fetchFromGitLab {
+    owner = "ternaris";
+    repo = "declinate";
+    rev = "v${version}";
```

```suggestion
    tag = "v${version}";
```

`tag` is usually preferred over `rev` to prevent accidentally fetching a branch with the same name